doc/sphinx/Pacemaker_Explained/constraints.rst

```diff
@@ -121,6 +121,23 @@ ___________________
        node (a value of **-INFINITY** changes "should" to "must"). A location
        constraint must either have a ``node`` and ``score``, or contain at
        least one rule.
+   * - .. rsc_location_role:
+       
+       .. index::
+          single: rsc_location; attribute, role
+          single: attribute; role (rsc_location)
+          single: role; rsc_location attribute
+       
+       role
+     - :ref:`enumeration <enumeration>`
+     - ``Started``
+     - The constraint applies only to resource instances in this role;
+       significant only for :ref:`promotable clones <s-resource-promotable>`.
+       Allowed values:
+       
+       * ``Started``: any active role
+       * ``Unpromoted``: the unpromoted role
```

Comment:
Aren't location constraints the ones where we treat `Started` and `Unpromoted` identically?
```
static bool
parse_location_role(const char *role_spec, enum rsc_role_e *role)
{
    ...
        case pcmk_role_started:
        case pcmk_role_unpromoted:
            /* Any promotable clone instance cannot be promoted without being in
             * the unpromoted role first. Therefore, any constraint for the
             * started or unpromoted role applies to every role.
             */
            *role = pcmk_role_unknown;
            break;
    ...
}
```

Reply:
Whoops, yes